### ISSUE TITLE
Suggestion: Add "user notes" for each game

### ISSUE THREAD

**Issue by asmaloney** [Heroic-Games-Launcher/HeroicGamesLauncher#2624]
### Problem description

I have lots of Windows games on EGS. Occasionally I will download one to try with the latest WINE to see if it works, but these downloads can be huge, so I don't always keep them around.

### Feature description

It would be cool to have a way to add notes to each game so I can track what versions I've tried (game & WINE), what issues I've run into, etc. - or any other notes I might want to keep about a game.

Maybe an editable "Notes" text area that fills the bottom of the main game page below the "Play Now" button?

### Alternatives

An external spreadsheet!

Maybe this is beyond the scope of Heroic, but having all the info in one place would be helpful.

**Comment by Etaash-mathamsetty** [comment]
is this what you are looking for (alternatives wise)?
https://docs.google.com/spreadsheets/d/1qvM4HpYLrDCi69nejKF3NIbqxr3RJZV3qQM9mL0u2Y0/edit#gid=0

**Comment by asmaloney** [comment]
Yes exactly. I already keep notes separately like that, but it would be nice to have some of that info right in the interface.

In case it's not clear from my original test, I'm not suggesting anything fancy. There doesn't need to be any structure - just a text box for a bunch of free-form text for me to write whatever I want about a given game. I would use it for my own testing info, others might use it for game review notes, or to track bugs for example.
